core/capabilities/remote/target/request/client_request.go

```diff
@@ -33,9 +34,10 @@ type ClientRequest struct {
 
 	respSent bool
 	mux      sync.Mutex
+	wg       *sync.WaitGroup
 }
 
-func NewClientRequest(ctx context.Context, lggr logger.Logger, req commoncap.CapabilityRequest, messageID string,
+func NewClientRequest(parentCtx context.Context, lggr logger.Logger, req commoncap.CapabilityRequest, messageID string,
```

Comment:
What is the significance of the name change?

Reply:
just to distinguish it from the the context created on line 62

Reply:
actually, the context passed into the client request is not being used now, the request is explicitly cancelled by the client, so there is no value to passing a context in.  So this is removed along with the bit of code that was creating a context object from the stop channel but leaking a goroutine as it wasn't being canclled.